python/sglang/srt/server.py

```diff
@@ -89,6 +89,22 @@
 tokenizer_manager = None
 
 
+@app.get("/v1/health")
+async def health(request: Request) -> Response:
+    """
+    Generate 1 token to verify the health of the inference service.
+    """
+    gri = GenerateReqInput(
+        text="s", sampling_params={"max_new_tokens": 1, "temperature": 0.7}
+    )
+    try:
+        _ = await tokenizer_manager.generate_request(gri, request).__anext__()
```

Comment:
We should be compatible with Python 3.8.

Reply:
Although we have manually upgraded Python to 3.10 in the Docker images we released, considering that some users are using Python 3.8 by default on Ubuntu 20.04, the `__anext__` used is only available after Python 3.10.

Reply:
```suggestion
        async for _ in tokenizer_manager.generate_request(gri, request):
            break
```

Reply:
Will using `async for _ in tokenizer_manager.generate_request(gri, request)` solve this issue?

I couldn't find a Python 3.8 sample in our repo, Did I miss something?

Reply:
Oh wow, that's exactly what I was thinking, lol.